components/clientComponents/forms/AddressComplete/utils.ts

```diff
@@ -0,0 +1,134 @@
+import { FormElement } from "@lib/types";
+import { AddressCompleteChoice, AddressCompleteResult, AddressElements } from "./types";
+import { Answer } from "@lib/responseDownloadFormats/types";
+
+const autoCompleteUrl =
+  "https://ws1.postescanada-canadapost.ca/AddressComplete/Interactive/AutoComplete/v1.00/json3.ws";
+const retriveAddressUrl =
+  "https://ws1.postescanada-canadapost.ca/AddressComplete/Interactive/RetrieveById/v1.00/json3.ws";
+
+// Function returns address complete list of choices.
```

Comment:
Have we thought about abuses for the calls to these APIs?

Reply:
They need a key, and are rate limited by CanadaPost on their end. There's not a lot more we can do to prevent people from hitting their endpoints. 

They're also just available for people to hit on the internet right now, and publicly documented.

In an ideal world, they'd provide a "Server Side" implementation that let us secure everything, but as is - this is just really a client side helper thing.